src/main/java/org/folio/service/CirculationRequestsRetriever.java

```diff
@@ -0,0 +1,64 @@
+package org.folio.service;
+
+import static org.folio.service.inventory.InventoryUtils.INVENTORY_LOOKUP_ENDPOINTS;
+import static org.folio.service.inventory.InventoryUtils.REQUESTS;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.folio.rest.core.RestClient;
+import org.folio.rest.core.models.RequestContext;
+import org.folio.rest.core.models.RequestEntry;
+
+import io.vertx.core.Future;
+
+public class CirculationRequestsRetriever {
+
+  private static final String REQUESTS_RECORDS = "requests";
+  private static final String REQUESTS_TOTAL = "totalRecords";
+  private static final String REQUEST_ID = "id";
```

Comment:
Also consider replacing with Enum
In Effective java suggest to use Enums over this type of patter.
https://medium.com/@swang354/effective-java-7357d032dd75
it is up to you. If you are going to leave as it was write some reasons to explain situation.